### ISSUE TITLE
rules_jsonnet is failing at Bazel@HEAD

### ISSUE THREAD

**Issue by sgowroji** [bazelbuild/rules_jsonnet#173]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/3301#018a8791-8198-403d-a216-2ec8c1b06bf3

Platform :  MacOs, Windows, Ubuntu

Logs:
```
ERROR: /private/var/tmp/_bazel_buildkite/6a57c906c53d89d0a6b15c856fdb49e5/external/io_bazel_rules_go/go/private/rules/transition.bzl:242:23: Use of Starlark transition without allowlist attribute '_allowlist_function_transition'. See Starlark transitions documentation for details and usage: @io_bazel_rules_go//go/private/rules:transition.bzl NORMAL
ERROR: error loading package '@google_jsonnet_go//cmd/jsonnet': at /private/var/tmp/_bazel_buildkite/a07a66f6e1e5d2c0dfbe5ca3818de267/external/io_bazel_rules_go/go/def.bzl:48:5: at /private/var/tmp/_bazel_buildkite/a07a66f6e1e5d2c0dfbe5ca3818de267/external/io_bazel_rules_go/go/private/rules/wrappers.bzl:34:5: initialization of module 'go/private/rules/transition.bzl' failed
(06:07:27) ERROR: /Users/buildkite/builds/bk-imacpro-5/bazel-downstream-projects/rules_jsonnet/jsonnet/BUILD:26:6: error loading package '@google_jsonnet_go//cmd/jsonnet': at /private/var/tmp/_bazel_buildkite/a07a66f6e1e5d2c0dfbe5ca3818de267/external/io_bazel_rules_go/go/def.bzl:48:5: at /private/var/tmp/_bazel_buildkite/a07a66f6e1e5d2c0dfbe5ca3818de267/external/io_bazel_rules_go/go/private/rules/wrappers.bzl:34:5: initialization of module 'go/private/rules/transition.bzl' failed and referenced by '//jsonnet:jsonnet_tool'
(06:07:27) ERROR: Analysis of target '//jsonnet:jsonnet_tool' failed; build aborted: Analysis failed

```

Steps:
```
git clone https://github.com/bazelbuild/rules_jsonnet.git && cd rules_jsonnet
git reset 7760d83db7e57961b2d865f75788b3b5f5db6ab8 --hard
export USE_BAZEL_VERSION=07e0d316a345a3cb2593f98525320590bbc56e30
 bazel build //... @examples//...
```
Culprit : https://github.com/bazelbuild/bazel/commit/10a46b1e85f0b5bf8564c125dd02cd4e70017452

CC Greenteam @meteorcloudy

**Comment by sgowroji** [comment]
Hi Team, Any update on this issue? PR  #174 is awaiting to merge. CC @[GuyMograbi](https://github.com/GuyMograbi). As the CI is failing from long time, We will be moving to disabled CI pipeline until its fixed.

**Comment by sgowroji** [comment]
Hi Team,  FYI We are moving this pipeline to disabled CI as the failures have persisted for a long time. Thanks!

**Comment by EdSchouten** [comment]
This should be fixed in #179, as we now use newer versions of jsonnet.
